api/models/Group.js

```diff
@@ -192,10 +187,15 @@ module.exports = bookshelf.Model.extend(merge({
     return this.get('num_members')
   },
 
-  async moderators () {
-    const responsibilities = await Responsibility.fetchForGroup(this.id)
-    const userIds = Responsibility.hasAllResponsibilities(responsibilities)
-    return this.members({ user_id: userIds }) // TODO RESP: this needs to be verified
+  // TODO: remove when mobile app has been updated
+  moderators () {
+    return this.stewards()
+  },
+
+  stewards () {
+    return this.members().query(q => {
+      q.whereRaw('exists (select * from group_memberships_common_roles inner join common_roles_responsibilities on common_roles_responsibilities.common_role_id = group_memberships_common_roles.common_role_id where common_roles_responsibilities.responsibility_id IN (1, 3, 4) and group_memberships_common_roles.user_id = users.id and group_memberships_common_roles.group_id = group_memberships.group_id)')
```

Comment:
oh responsibilities got defined ids?

Reply:
ah yes, this is one of my main ways to try and improve performance. yes common roles and responsibilities all get created in the initial migration after the table gets created so they will always have these ids. this means we dont need the extra table join